packages/commons/src/events/FieldConfig.ts

```diff
@@ -322,7 +322,12 @@ export type Office = z.infer<typeof Office>
 
 const Address = BaseField.extend({
   type: z.literal(FieldType.ADDRESS),
-  defaultValue: AddressFieldValue.optional()
+  defaultValue: AddressFieldValue.optional(),
+  configuration: z
+    .object({
+      defaultsToUserPrimaryOfficeLocation: z.boolean()
```

Comment:
```suggestion
      defaultToUserDistrict: z.boolean()
```

Could this be more descriptive?